curriculum/challenges/english/25-front-end-development/lab-confidential-email-page/66bba6fff611169359d9d36a.md

```diff
@@ -0,0 +1,249 @@
+```
+
+Your `#confidential` element should have its `display` set to `inline-block`.
+
+```js
+assert.equal(window.getComputedStyle(document.querySelector('#confidential'))?.display, 'inline-block');
+```
+
+Your `#top-secret` element should have its `display` set to `inline-block`.
+
+```js
+assert.equal(window.getComputedStyle(document.querySelector('#top-secret'))?.display, 'inline-block');
+```
+
+Your `#confidential` element should have a `padding` on all sides.
+
+```js
+const style = window.getComputedStyle(document.querySelector('#confidential'));
+assert.notEqual(style?.paddingTop, '0px');
+assert.notEqual(style?.paddingRight, '0px');
+assert.notEqual(style?.paddingBottom, '0px');
+assert.notEqual(style?.paddingLeft, '0px');
+```
+
+Your `#top-secret` element should have a `padding` on all sides.
+
+```js
+const style = window.getComputedStyle(document.querySelector('#top-secret'));
+assert.notEqual(style?.paddingTop, '0px');
+assert.notEqual(style?.paddingRight, '0px');
+assert.notEqual(style?.paddingBottom, '0px');
+assert.notEqual(style?.paddingLeft, '0px');
+```
+
+Your `#confidential` element should have a `left-margin`.
+
+```js
+assert.notEqual(window.getComputedStyle(document.querySelector('#confidential'))?.marginLeft, '0px');
+```
+
+Your `#top-secret` element should have a `left-margin`.
+
+```js
+assert.notEqual(window.getComputedStyle(document.querySelector('#top-secret'))?.marginLeft, '0px');
+```
+
+Your `#confidential` element should have a `border`.
+
+```js
+assert.notEqual(window.getComputedStyle(document.querySelector('#confidential'))?.borderWidth, '0px');
+```
+
+Your `#top-secret` element should have a `border`.
+
+```js
+assert.notEqual(window.getComputedStyle(document.querySelector('#top-secret'))?.borderWidth, '0px');
+```
+
+Your `#confidential` element should use a CSS `transform` to rotate the element.
+
+```js
+assert.notEqual(window.getComputedStyle(document.querySelector('#confidential'))?.transform, 'none');
+```
+
+Your `#top-secret` element should use a CSS `transform` to rotate the element.
+
+```js
+assert.notEqual(window.getComputedStyle(document.querySelector('#top-secret'))?.transform, 'none');
+```
+
+You should have at least three paragraph elements within your `#email` element.
+
+```js
+assert.isAtLeast(document.querySelectorAll('#email > p').length, 3);
+```
+
+You should have at least three `span` elements with a class of `blurred` within your paragraphs.
+
+```js
+assert.isAtLeast(document.querySelectorAll('p > span.blurred').length, 3);
+```
+
+Your `.blurred` elements should not be empty.
+
+```js
+const els = document.querySelectorAll('.blurred');
+els.forEach(el => assert.isAtLeast(el.innerText.length , 1))
+```
+
+You should have a `.blurred` selector that set its elements `filter` to `blur(3px)`.
+
+```js
+assert.equal(new __helpers.CSSHelp(document).getStyle('.blurred')?.filter, 'blur(3px)');
+```
+
+# --seed--
+
+## --seed-contents--
+
+```html
+
```

Comment:
Asking again because I'm not sure what we decided. Are we going to provide the boilerplate?